libs/pages/services/src/lib/feature/page-application-create-feature/page-application-create-general-feature/page-application-create-general-feature.tsx

```diff
@@ -48,7 +59,20 @@ export function PageApplicationCreateGeneralFeature() {
   }, [watchServiceType, dispatch])
 
   const onSubmit = methods.handleSubmit((data) => {
-    setGeneralData(data)
+    const cloneData = {
+      ...data,
+    }
+
+    if (data.serviceType === ServiceTypeEnum.CONTAINER && data.cmd_arguments) {
+      cloneData.cmd = eval(data.cmd_arguments)
```

Comment:
I am not sur to understand why you use eval here ? You need to calculate addition on string ?
Apparently, you have to use eval wisely otherwise could be a risk 
[https://www.w3schools.com/jsref/jsref_eval.asp](https://www.w3schools.com/jsref/jsref_eval.asp
)